### ISSUE TITLE
unsubstituted parameter doesn't cause an error

### ISSUE THREAD

**Issue by kireet** [feedly/transfer-nlp#23]
something like this won't cause a problem:
```
{ 
   "item": {
       "_name": "foo",
       "param":"$bar"
    }
}
```

even if we don't set a value for `bar`. this can lead to easily misconfigured objects.

**Comment by petermartigny** [comment]
if `foo` is a registered class, this will search for an existing `bar` object with the 3 different `default_params_mode`s, and if it doesn't find it in the end it raises an `UnconfiguredItemsException` error

**Comment by petermartigny** [comment]
Resolved with PR #26  :
 https://github.com/feedly/transfer-nlp/commit/c89cc7a151dc8806fac9aa28799ce394f5c561c9
